solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -346,6 +346,25 @@ def _legend_text(name, max_length=20):
         return name
 
 
+def formatted_interval(interval):
+    """Converts an interval_length timedelta into a string for display
+
+    Parameters
+    ----------
+    minutes: np.timedelta64
+
+    Returns
+    -------
+    str
+        The interval as a string, displayed in the largest units possible
+        without mixing units(up to days)
+    """
+    if (interval % np.timedelta64(1, 'h') == 0):
+        return f'{np.timedelta64(interval, "h").astype(int)}h'
+    else:
+        return f'{np.timedelta64(interval, "m").astype(int)}m'
```

Comment:
Is there a reason to prefer `np.timedelta64` to `pd.Timedelta` here? I have a lot more experience with the pandas functions and generally don't trust the numpy ones as much. And we use the pandas versions throughout SFA.

I had to remind myself that the relevant data is in the `components` attribute:

```
>>> pd.Timedelta('1h').components.hours
1
```

Reply:
I have no idea why, but the metadata Dataframe contains the timedeltas. `_fill_timeseries` also expects the numpy timedeltas. I'd definitely prefer pandas timedeltas here as well but I don't know  of a simple way to make that conversion.

Reply:
Ok, good to know. If we ever change that type in the DataFrame then we'll need to change this since the `if` statement will no longer work

```
>>> pd.Timedelta('1h') % np.timedelta64(1, 'h')
Timedelta('0 days 00:00:00')
>>> pd.Timedelta('1h') % np.timedelta64(1, 'h') == 0
False
```